### ISSUE TITLE
Developer Guides: Sidedness

### ISSUE THREAD

**Issue by Android789515** [FabricMC/fabric-docs#230]
https://docs.fabricmc.net/develop/items/first-item#preparing-your-items-class This section here does not specify where to make the `ModItems` class.  Whether in main or client folder.

---

### Maintainer notes:

- [ ] Create new "sidedness" page that goes through client, server, integrated servers, etc. - A spiritual improvement of the wiki's sidedness page.

**Comment by Linguardium** [comment]
Items would be common since both logical client (rendering) and logical server (logic) needs to know about them. The differences in client and main should have been covered in a previous section of the docs

**Comment by Android789515** [comment]
It could be elaborated more.  It just says

The src/main/java folder is used to store the Java source code for your mod - it exists on both the client and server environments.

**Comment by Linguardium** [comment]
Agreed. the logical server/logical client concept should be elaborated on the docs. it has not yet replaced the wiki page on it https://wiki.fabricmc.net/tutorial:side

**Comment by JR1811** [comment]
I created a (very high level) overview on that some time ago, if someone wants to use it on a potential PR

![Image](https://github.com/user-attachments/assets/be9cc475-6e63-489d-94ae-ef7e92c75e6b)

**Comment by cassiancc** [comment]
Prior art also includes https://github.com/natanfudge/fabric-docs/blob/master/docs/modding-tutorials/conventions-and-terminology/side.md
